src/rules/converters/variable-name.ts

```diff
@@ -0,0 +1,116 @@
+import { RuleConverter } from "../converter";
+
+export const convertVariableName: RuleConverter = tslintRule => {
+    const getCamelCaseRuleOptions = () => {
+        const camelCaseOptionNotice: string[] = [];
+        camelCaseOptionNotice.push(
+            "By default, this rule looks for any underscores (_) located within the source code. It ignores leading and trailing underscores and only checks those in the middle of a variable name.",
```

Comment:
Phrasing suggestion, since the notices are intended to inform the user of _differences_ between linter behaviors...

```suggestion
            "Leading and trailing underscores (_) in variable names will now be ignored.",
```

However, TSLint's `variable-name` rule has `allow-leading-underscore` and `allow-trailing-underscore` options, so there are a few cases for this notice:

* If both of those options are enabled, this notice shouldn't be added
* If one option is enabled, it should only say that type: e.g. _"Leading undescores in variable names ..."_
* If both options are enabled, this suggested message is :+1:

Reply:
The notice should be displayed if **both** options are enabled or not?

Reply:
If both options are enabled the notice should not be displayed. ESLint's rule behaves the same as the TSLint rule with both options enabled (right?).